server/matchmaker/search.py

```diff
@@ -123,6 +147,17 @@ def match(self, other: 'Search'):
         :return:
         """
         self._logger.info("Matched %s with %s", self.player, other.player)
+        
+        numgames = self.player.numGames
+        if numgames <= config.NEWBIE_MIN_GAMES:
```

Comment:
```suggestion
        if numgames == config.NEWBIE_MIN_GAMES:
```
why <= instead of < ?
If a newbie has a the min amount of games doesn't that imply he isn't one anymore?

Reply:
This was in the original code, so I assume it is intended behavior from Exotic_retard. The newbie must have "more than X games".